chemprop/train/run_training.py

```diff
@@ -149,7 +154,12 @@ def run_training(args: TrainArgs,
     # Initialize scaler and scale training targets by subtracting mean and dividing standard deviation (regression only)
     if args.dataset_type == 'regression':
         debug('Fitting scaler')
-        scaler = train_data.normalize_targets()
+        if args.is_atom_bond_targets:
+            scaler = None
+            atom_bond_scalers = train_data.normalize_targets()
```

Comment:
Oh I see, on the other side you're giving the list of scalers a different variable name. That could work.